src/plugin.py

```diff
@@ -452,6 +452,22 @@ def _get_not_added_free_games(owned_games):
         log.info("Plugin shutdown.")
         await self.authentication_client.shutdown()
 
+    async def get_game_time(self, game_id, context):
+        if self.local_client.load_local_files_done:
+            for game in self.local_client.config_parser.games:
+                try:
+                    blizzard_game = Blizzard[game.uid]
+                except KeyError:
+                    log.warning(f'[{game.uid}] is not known blizzard game. Skipping')
+                    return None
+
+                if blizzard_game.id == game_id:
+                    last_played = int(game.last_played)
+                    log.debug(f'get_game_time return {game_id} {last_played}')
+                    return GameTime(game_id, None, last_played_time=last_played)
+
+        log.warning(f'warning get_game_time failed for {game_id}')
```

Comment:
what about:
```python
last_played = None
for game in self.local_client.config_parser.games:
     ...
     if game.last_played is not None:
         last_played = int(game.last_played)
         break

return GameTime(game_id, None, last_played_time=last_played)
```

This would be analogous to #7

Reply:
@bartok765 Yes , need some merge work.